doc/man1/flux-cron.rst

```diff
@@ -6,182 +6,262 @@ flux-cron(1)
+  Set a name for this cron entry to *STRING*.
 
-.. program:: flux cron at
+.. option:: -a, --after=TIME
+
+  The first task will run after a delay of *TIME* instead of *interval*.
+  After the first task the entry will continue to execute every *interval*.
+
+.. option:: -c, --count=N
+
+  The entry will be run a total of *N* times, then stopped.
 
 .. option:: -o, --options=LIST
 
-   Set comma separated EXTRA OPTIONS for all cron entries.
+  Set comma separated `EXTRA OPTIONS`_ for this cron entry.
 
 .. option:: -E, --preserve-env
 
-   The :option:`--preserve-env` option allows the current environment to be
-   exported and used for the command being executed as part of the cron job.
-   Normally, the broker environment is used.
+  Export the current environment to be used for the command being executed
+  as part of the cron job.  Normally, the broker environment is used.
 
 .. option:: -d, --working-dir=DIR
 
-   The :option:`--working-dir` option allows the working directory to be set
-   for the command being executed as part of the cron job. Normally, the
-   working directory of the broker is used.
+  Set the working directory to be set for the command being executed as part
+  of the cron job. Normally, the working directory of the broker is used.
+
+
+list
+----
+
+.. program:: flux cron list
+
+Display a list of current entries registered with the cron module and
+their current state, last run time, etc.
+
+stop
+----
+
+.. program:: flux cron stop
+
+Stop cron entry *id*. The entry will remain in the cron entry list until
+deleted.
+
+start
+-----
+
+.. program:: flux cron start
+
+Start a stopped cron entry *id*.
+
+delete
+------
 
-**list**
-   Display a list of current entries registered with the cron module and
-   their current state, last run time, etc.
+.. program:: flux cron delete
 
-**stop** *id*
-   Stop cron entry *id*. The entry will remain in the cron entry list until
-   deleted.
+Purge cron entry *id* from the cron entry list.
 
-**start** *id*
-   Start a stopped cron entry *id*.
+.. option:: -k, --kill
 
-**delete** [--kill] *id*
-   Purge cron entry *id* from the cron entry list. If :option:`--kill` is
-   used, kill any running task associated with entry *id*.
+Kill any running task associated with entry *id*.
 
-**dump** [--key=KEY] *id*
-   Dump all information for cron entry *id*. With :option:`--key` print only
-   the value for key *KEY*. For a list of keys run
-   :program:`flux cron dump ID`.
+dump
+----
+
+.. program:: flux cron dump
+
+Dump all information for cron entry *id*.
+
+.. option:: -k, --key=KEY
+
+Print only the value for key *KEY*.
+
+For a list of keys run :option:`flux cron dump ID`.
+
+sync
+----
+
+.. program flux cron sync
+
+Query and modify the current **sync-event** behavior for the cron module.
+If a sync-event is set, the cron module will defer all task execution
+until an event matching the sync-event *topic* is received.
```

Comment:
I feel just a bit more would be useful.  Maybe mention is used for ... managing crons so they limit interruption of tasks?   sometimes used with `flux-event`?